src/lib/components/tour-overlay/TourOverlay.component.tsx

```diff
@@ -152,38 +107,68 @@ export const TourOverlay = forwardRef<TourOverlayRef, TourOverlayProps>((props,
   }, [tourStep, onBackdropPress, current, goTo, next, previous, start, stop]);
 
   useEffect(() => {
-    const moveSpot = Animated.parallel([
-      Animated.spring(center, {
-        damping: 50,
-        mass: 5,
-        stiffness: 300,
-        toValue: { x: cx, y: cy },
-        useNativeDriver,
-      }),
-      Animated.spring(radius, {
-        damping: 30,
-        mass: 5,
-        stiffness: 300,
-        toValue: r,
-        useNativeDriver,
-      }),
-      Animated.timing(tipOpacity, {
-        delay: 500,
-        duration: 300,
+    tooltipRef.current?.measureInWindow((_x, _y, width, height) => {
+      setTooltipStyle(() => {
+        const align = tourStep.alignTo ?? Align.SPOT;
+        const half = (Math.max(spot.width, spot.height) / 2) * 1.15;
```

Comment:
What is 1.15, is it PI? Could we use the `Math` const or lets put it as a constant to know what it represents.

Reply:
The `1.15` multiplication adds 15% to the `half` const. The idea is to increase the area surrounding the element to add a gap between the spotlight and the tooltip. We do the same when we calculate the size of the spotlight, so it's a great idea to move it to a constant. Good catch! 🙂